### ISSUE TITLE
A Simple Log based Loss Function for Ordinal Text Classification

### ISSUE THREAD

**Issue by RoyiAvital** [carlomarxdk/cdw-cross-entropy-loss#1]
Hi,

This method is highly similar to [A Simple Log based Loss Function for Ordinal Text Classification](https://github.com/glanceable-io/ordinal-log-loss) with the loss:

$$
\mathcal{L}_{\alpha} \left( P, y \right) = - {\sum}_{i = 1}^{N} \log \left( 1 - {p}_{i} \right) {d \left( y, i \right)}^{\alpha}
$$

![png](https://github.com/carlomarxdk/cdw-cross-entropy-loss/assets/7347975/4571dbb1-9d01-4019-b17b-b3d7d2535937)

So the above generalizes what's here.
Nice to see you both came up with something similar.
